doc/conf.py

```diff
@@ -83,7 +83,9 @@
     'extra_nav_links': {
         'GitHub': 'https://github.com/jnikula/hawkmoth',
         'PyPI': 'https://pypi.org/project/hawkmoth',
-    }
+    },
+    'sidebar_width': '280px',
+    'page_width': '1000px',
```

Comment:
Are hidpi screens handled automatically through scaling? I think not, but bear in mind I know very little of CSS.

Reply:
Not my strong suit either, and I can only justify this because the original Alabaster stylesheet uses px.